Sources/Plasma/Apps/plClient/Mac-Cocoa/MainMenu.xib

```diff
@@ -0,0 +1,679 @@
+                    <modifierMask key="keyEquivalentModifierMask"/>
+                    <menu key="submenu" title="Uru Live" systemMenu="apple" id="uQy-DD-JDr">
+                        <items>
+                            <menuItem title="About Uru Live" id="5kV-Vb-QxS">
+                                <modifierMask key="keyEquivalentModifierMask"/>
+                                <connections>
+                                    <action selector="orderFrontStandardAboutPanel:" target="-1" id="Exp-CZ-Vem"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem isSeparatorItem="YES" id="VOq-y0-SEH"/>
+                            <menuItem title="Preferences…" keyEquivalent="," id="BOF-NM-1cW"/>
+                            <menuItem isSeparatorItem="YES" id="wFC-TO-SCJ"/>
+                            <menuItem title="Services" id="NMo-om-nkz">
+                                <modifierMask key="keyEquivalentModifierMask"/>
+                                <menu key="submenu" title="Services" systemMenu="services" id="hz9-B4-Xy5"/>
+                            </menuItem>
+                            <menuItem isSeparatorItem="YES" id="4je-JR-u6R"/>
+                            <menuItem title="Hide Uru Live" keyEquivalent="h" id="Olw-nP-bQN">
+                                <connections>
+                                    <action selector="hide:" target="-1" id="PnN-Uc-m68"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Hide Others" keyEquivalent="h" id="Vdr-fp-XzO">
+                                <modifierMask key="keyEquivalentModifierMask" option="YES" command="YES"/>
+                                <connections>
+                                    <action selector="hideOtherApplications:" target="-1" id="VT4-aY-XCT"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem title="Show All" id="Kd2-mp-pUS">
+                                <modifierMask key="keyEquivalentModifierMask"/>
+                                <connections>
+                                    <action selector="unhideAllApplications:" target="-1" id="Dhg-Le-xox"/>
+                                </connections>
+                            </menuItem>
+                            <menuItem isSeparatorItem="YES" id="kCx-OE-vgT"/>
+                            <menuItem title="Quit Uru Live" keyEquivalent="q" id="4sb-4s-VLi">
+                                <connections>
+                                    <action selector="terminate:" target="-1" id="Te7-pn-YzF"/>
+                                </connections>
+                            </menuItem>
+                        </items>
+                    </menu>
+                </menuItem>
+                <menuItem title="File" id="dMs-cI-mzQ">
```

Comment:
The File, Edit, Format, etc. menus seem like they probably don't make too much sense for a game client?

Reply:
At least a minimal Edit menu should exist for every application that has text fields, which we do on the login screen. Agreed though that all the document management and text formatting menus/items aren't needed here. I would look at macOS' own Archive Utility as an example for which menus are needed in an application with not much UI.

Reply:
File and Edit both need to exist. (For copy/paste like noted above, File needs to be there for Quit.) Window menu generally needs to be there as well. It holds the system full screen transition.

I can take a look at stuff like Format and pair things down some more.

Reply:
Gah - I forgot quit is in the application menu now. (Apparently my brain is still Classic macOS.) Lemme see what the HIG says about removing file.

Reply:
> If your app doesn’t handle any types of files, you can rename or eliminate this menu.

Boom. I'll take it out.

Reply:
> Gah - I forgot quit is in the application menu now. (Apparently my brain is still Classic macOS.)

I thought the same thing; totally forgot even though I use it in the modern location all the time.

Reply:
Ok, I've removed File and a few more menu items.

I've left Edit intact. As mention by @dgelessus - removing this menu entire will break things like copy/paste. And as someone who uses copy/paste on the login screen sometimes this would greatly annoy me. Arguably the edit menu could be paired down - but it won't hurt anything leaving it in the default config. Menu items that aren't applicable will just be disabled.